recipe/meta.yaml

```diff
@@ -1,38 +1,56 @@
-{% set version = "3.0.0b2.post2" %}
+{% set version = "3.0.0b2" %}
+{% set version_str = "v3.0.0-beta-2" %}
 
 package:
     name: protobuf
     version: {{ version }}
 
 source:
-    fn: protobuf-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/p/protobuf/protobuf-{{ version }}.tar.gz
-    md5: 42aa7ab79ebcb94917e938ff482ef7a6
+    fn: protobuf_{{ version }}.tar.gz
+    url: https://github.com/google/protobuf/archive/{{ version_str }}/protobuf-{{ version_str }}.tar.gz
+    md5: e7f2602baffcbc27fb607de659cfbab6
 
 build:
-    number: 2
+    number: 3
+    skip: true  # [win]
 
 requirements:
     build:
+        - autoconf
+        - automake
+        - libtool
+        - pkg-config
         - python
-        - setuptools
+        - six
+        - ordereddict        # [py26]
+        - unittest2          # [py26]
+        - gcc                # [osx]
```

Comment:
Right, hence the "maybe it's 10.9 that has the problem". Also seemed like adding a compiler flag was a hell of a lot simpler workaround than including an entire additional toolchain.

At any rate, it's not worth more energy on the discussion--so long as the packages work on my 10.11 box. (This currently isn't always the case: conda-forge/netcdf4-feedstock#4.)